Version 2.0.0

joseconti opened the pull request.

alexandresaiz commented:
> Please add `.gitignore` and remove .DS_Store files

@jimmyn if you don't mention @joseconti won't probably know he needs to do sth

joseconti commented:
Ok, Thank you @jimmyn

joseconti commented:
All done @jimmyn 

* All .DS_Store removed
* Added .gitignore
* Added Refunds
* Updated readme.txt changelog
* Declared compatibility with WooCommerce 4.1
